recipe/bld.bat

```diff
@@ -0,0 +1,20 @@
+setlocal EnableDelayedExpansion
+
```

Comment:
```suggestion
setlocal EnableDelayedExpansion

```
`EnableDelayedExpansion` is useless in this case since you don't use the `!VAR!` special syntax.

Reply:
Thanks!